template/YYYY-MM-feature-spec-template.md

```diff
@@ -0,0 +1,74 @@
+# Topic: TODO
+
+* **Status**: Pending/Approved
+* **Author**: Your name (@YourGitHubUserName)
+
+## Topic Summary
+<!-- A paragraph or two to summarize the topic area. Just define it in summary form so we all know what it is. -->
+
+### Top level goals
+<!-- At the most basic level, what are we trying to accomplish? -->
+
+### Non-goals (out of scope)
+<!-- What are we explicitly not trying to accomplish? -->
+
+## Customer profile and challenges
+<!-- Define the primary customer and the key problem / pain point we intend to address for that customer. If there are multiple customers or primary and secondary customers, call them out.   -->
+
+### Customer persona(s)
+<!-- Who is the target customer? Include size/org-structure/decision makers where applicable. -->
+
+### Challenge(s) faced by the customer
+<!-- What challenges do the customer face? Why are they experiencing pain and why do current offerings not meet their need? -->
+
+### Positive customer outcome
+<!-- What is the positive outcome for the customer if we deliver this, i.e. what is the value proposition for the customer? Remember, this is customer-centric. -->
+
+## Key scenarios
+<!-- List ~3-7 high level scenarios to clarify the value and point to how we will decompose this big area into component capabilities. We may ultimately have more than one level of scenario. -->
+
+### Scenario 1: TODO
+<!-- One or two sentence summary -->
+
+### Scenario 2: TODO
+<!-- One or two sentence summary -->
+
+### Scenario 3: TODO
+<!-- One or two sentence summary -->
+
+## Key dependencies and risks
+<!-- What dependencies must we take in order to enable this scenario? -->
+<!-- What other risks are you aware of that need to be mitigated. If you have a mitigation in mind, summarize here. -->
+<!-- **Dependency Name** – summary of dependency.  Issues/concerns/risks with this dependency -->
+<!-- **Risk Name** – summary of risk.  Mitigation plan if known. If it is not yet known, no problem. -->
+
+## Key assumptions to test and questions to answer
+<!-- If you are making assumptions that, if incorrect, would cause us to significantly alter our approach to this scenario, make them explicit here.  Also call out how / when you plan to validate key assumptions. -->
+<!-- What big questions must we answer in order to clarify our plan for this scenario.  When and how do you plan to answer those questions (prototype feature x, customer research, competitive research, etc) -->
```

Comment:
```suggestion
## Key assumptions to test and questions to answer
<!-- If you are making assumptions that, if incorrect, would cause us to significantly alter our approach to this scenario, make them explicit here.  Also call out how / when you plan to validate key assumptions. -->
<!-- What big questions must we answer in order to clarify our plan for this scenario.  When and how do you plan to answer those questions (prototype feature x, user research, competitive research, etc.) -->
```